docs/.vitepress/config.ts

```diff
@@ -162,6 +162,26 @@ export default defineConfig({
             { text: 'Mouse Bindings', link: '/reference/mouse_bindings.md' },
             { text: 'Menus', link: '/reference/menus.md' },
             { text: 'Metadata', link: '/reference/metadata.md' },
+            {
+              text: 'Package Control',
+              items: [
+                { text: 'Channels and Repositories', link: '/reference/package-control/channels_repositories.md' },
```

Comment:
Do you want to rename this one to kebab case, too? Or is there a reason you kept it snake case?

Reply:
I highly doubt I had a good reason for that 😂